sass/layout/_navigation.scss

```diff
@@ -201,4 +201,14 @@
     }
 }
 
-
+.app__body {
+    #navbar_wrapper {
+        .navbar-default {
+            .navbar-toggle{
+                .icon-bar{
+                    background: var(--sidebar-header-text-color);
```

Comment:
looks like there is an existing selector on line 107: https://github.com/mattermost/mattermost-webapp/pull/6720/files#diff-33df6171c54ae8787ddb1c33f38d4496R107

Maybe lets move this there for better maintainability.

Reply:
@ali-farooq0, thanks for the review. done!